release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -624,6 +624,15 @@ For more information, see xref:../scalability_and_performance/node-observability
 [id="ocp-4-11-auth"]
 === Authentication and authorization
 
+[id="ocp-4-11-post-installation"]
+=== Post-installation configuration
+
+[id="ocp-4-11-cluster-capabilities"]
+==== Cluster capabilities
+As a cluster administrator, you can enable cluster capabilities to select or deselect one or more optional components before installation or post installation.
+
+For more information, see Cluster capabilities.
```

Comment:
Can you add an xref here? We are allowed (and encouraged) to add xrefs in release notes sections